src/pages/MyPage/MyPage.js

```diff
@@ -1,3 +1,137 @@
+import styled from 'styled-components';
+import { BiDotsVerticalRounded } from 'react-icons/bi';
+
 export default function MyPage() {
-  return <h1>MyPage 입니다.</h1>;
+  return (
+    <ResumeContainer>
+      <ResumeMakeBox>새 이력서 작성</ResumeMakeBox>
+      <ResumeMakeBox>파일 업로드</ResumeMakeBox>
+      {RESUME_LIST.map(({ userId, resumeId, title, hasCompleted }) => {
+        return (
+          <Resume key={resumeId} userId={userId}>
+            <ResumeTop>
+              <ResumeTopButton name="matchUp">매치업 이력서</ResumeTopButton>
+              <ResumeTopButton name="career">경력 인증</ResumeTopButton>
+            </ResumeTop>
+            <ResumeMiddle>
+              <ResumeMiddleTextBox title>
+                FE 개발자 임채동입니다.
+              </ResumeMiddleTextBox>
+              <ResumeMiddleTextBox>2022.08.29</ResumeMiddleTextBox>
+            </ResumeMiddle>
+            <ResumeBottom>
+              <ResumeBottomLang>한</ResumeBottomLang>
+              <div>작성 완료</div>
+              <div>
+                <BiDotsVerticalRounded />
+              </div>
+            </ResumeBottom>
+          </Resume>
+        );
+      })}
+    </ResumeContainer>
+  );
 }
+
+const ResumeContainer = styled.div`
+  ${props => props.theme.variables.flex()}
+  flex-wrap : wrap;
+  width: 50rem;
+`;
+
+const Resume = styled.div`
+  ${props =>
+    props.theme.variables.flex('column', 'space-around', 'space-between')}

```

Comment:
```suggestion
  ${({theme}) =>theme.variables.flex('column', 'space-around', 'space-between')}
```
매개변수자리에서 바로 구조분해할당 가능합니다.
반복되는 `props` 를 줄일 수 있습니다